projects/ngneat/transloco/src/lib/transloco.transpiler.ts

```diff
@@ -72,6 +80,19 @@ export class DefaultTranspiler implements TranslocoTranspiler {
   }
 }
 
+function getInterpolationRegExpFromConfig(userConfig?: TranslocoConfig): RegExp {
+  const interpolation = userConfig ? userConfig.interpolation : defaultConfig.interpolation;
```

Comment:
Looking at the code now, I don't think we need to support Regex, don't see any reason for someone to pass a regex instead of the markings. Do you agree?

Update the default value to: `['{{', '}}']`
```
const [start, end] = userConfig?.interpolation ?? defaultConfig.interpolation;

return new RegExp(`${start}(.*?)${end}`, 'g');
```

Reply:
Makes sense to me to remove the RegExp 👍  

Sadly we can't use optional chaining and nullish coalescing as this project is still on typescript 3.2